README.md

```diff
@@ -183,6 +184,8 @@ Several parts of the provided schema have been defined (DrawingRules, LabelRules
 
 Find a bug or want to request a new feature?  Please let us know by submitting an issue.
 
+Issues with the Esri-supported stylx files are not managed in this repository. Please contact [Esri Support Services](https://support.esri.com/en/contact-tech-support) or log them in [Esri/military-symology](https://github.com/Esri/military-symbology).
```

Comment:
`Issues with the Esri-supported stylx files are not managed in this repository. Please contact [Esri Support Services](https://support.esri.com/en/contact-tech-support) or log them in [Esri/military-symology](https://github.com/Esri/military-symbology).`

Need to add a 'b' to the [Esri/military-symology](https://github.com/Esri/military-symbology)